drivers/display/Kconfig.sdl

```
@@ -39,4 +39,10 @@ choice SDL_DISPLAY_DEFAULT_PIXEL_FORMAT
 
 endchoice
 
+config SDL_DISPLAY_ZOOM
+        int "Default zoom"
+        default 1
```

Perhaps this should be expressed as a percentage so it is possible to zoom 150%?

```suggestion
        int "Default zoom"
        default 100
```
	*window = SDL_CreateWindow("Zephyr Display", SDL_WINDOWPOS_UNDEFINED,
				   SDL_WINDOWPOS_UNDEFINED, width * CONFIG_SDL_DISPLAY_ZOOM / 100,
				   height * CONFIG_SDL_DISPLAY_ZOOM / 100, SDL_WINDOW_SHOWN);
```c

```

---

So you suggest renaming the option `CONFIG_SDL_DISPLAY_ZOOM_PERCENTAGE` to make it explicit?

---

Sure, or `_PCT` if you prefer it shorter. The option handle and description should make this clear.